src/main/api/2/studio-api-2.yaml

```diff
@@ -6168,6 +6295,36 @@ components:
                 type: string
                 description: The adjusted value if available
 
+    ClipboardItem:
+      type: object
+      properties:
+        path:
+          type: string
+          description: The path of the item
+        children:
+          type: array
+          description: The child items
+          items:
+            $ref: '#/components/schemas/ClipboardItem'
+      required:
+        - path
```

Comment:
Are `children` meant to be the child items that were selected to be copied too?

This hierarchical structure makes it a bit complex to build the request, plus the request body will be larger. Would simply having a list of paths not be sufficient? Paths bear hierarchy on their own.

Reply:
This API was modeled following the existing one, there is a lot of logic on the backend that already expects this structure